src/parser/open-api/v3/open-api-v3-parser.js

```diff
@@ -230,7 +233,7 @@ function getUIParametersForLocation (parameters, location) {
 function getUIRequest (description, requestBody = null) {
   const uiRequest = {}
 
-  if (description) {
+  if (description && description !== '') {
```

Comment:
I thought an empty string would evaluate to false, as in https://developer.mozilla.org/en-US/docs/Glossary/Falsy?